cmd/migrations/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+
+	"github.com/go-devs-ua/octagon/app/repository/pg"
+	"github.com/go-devs-ua/octagon/cfg"
+	"github.com/go-devs-ua/octagon/lgr"
+	_ "github.com/lib/pq"
+
+	migrate "github.com/rubenv/sql-migrate"
+)
+
+func main() {
+	logger := lgr.New()
+
+	opt, err := cfg.GetConfig()
+	if err != nil {
+		logger.Errorf("Failed to get config from .env: %+v\n", err)
+		return
+	}
+
+	db, err := pg.ConnectDB(opt)
+	if err != nil {
+		logger.Errorf("%+v\n", err)
+		return
+	}
+
+	direction := flag.String("migrate", "", "applying migrations 'up/down'")
```

Comment:
What happen when you receive not up/down argument value?

Reply:
I assume that we use this main only when we need to use migrations. So default situation will be that we using migration up...

Reply:
I agree with you but... it would be much better if we return an error in case a wrong value is received.

If i give you a wrong payload you should tell me this but nut just ignore it.

Reply:
But is simple if statement will be enough to handle this cases or should it be - i don't know - more elegant way?)

Reply:
Add if statement. Fixed.